Docs/deployment/configuration.md

```diff
@@ -0,0 +1,118 @@
+# Configuration
+
+Two configuration methods can be used to control Santa: local configuration and a sync server controlled configuration.
```

Comment:
I'm not entirely sure how to word it but this isn't 100% accurate - even with a sync server some configuration is only available locally (we don't sync down any of the GUI options, though maybe we should).